### ISSUE TITLE
View Messages posted on topics?

### ISSUE THREAD

**Issue by jigar1859** [yahoo/CMAK#903]
This might be the lamest question; however, I am unable to figure this out, can someone guide me?

I have added my cluster
Able to create topics form the CMAK UI and Java Code.
Able to post messages to my topics

How do I view those messages on the CMAK UI?

![image](https://user-images.githubusercontent.com/1884314/216847014-3daf4106-21de-451d-8e6d-659728e22aca.png)

**Comment by OneCricketeer** [comment]
I dont believe this tool has ever had a topic browser. AKHQ, Kafdrop, and KafkaUI do, however.
